sdk/storage/src/table/table_context.rs

```diff
@@ -0,0 +1,4 @@
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct TableContext {}
```

Comment:
What is this for?

Reply:
In the creation of `PipelineContext`, ` TableContext ' is the second parameter. At the moment I have no data that needs to be passed between pipeline policies.
Is there a way to create `PipelineContext` without any `Content: C` ?

Reply:
@ShayMusachanov-dev we are actively discussing how to handle pipeline options in #474. The current solution leaves much to be desired as you have noticed here. 
I would not fret too much here now as it's bound to change soon. 

One particular point I think should be discussed is if we should pass the `AuthorizationPolicy` down the pipeline instead of embedding it in the policy (as you here like it's currently done in CosmosDB). While the current approach is certainly easier, it prevents the user to change the authorization for a specific call, if need be.